tesseract_collision/core/include/tesseract_collision/core/types.h

```diff
@@ -485,6 +497,9 @@ struct CollisionCheckConfig
   /** @brief Secifies the mode used when collision checking program/trajectory. Default: ALL */
   CollisionCheckProgramType check_program_mode{ CollisionCheckProgramType::ALL };
 
+  /** @brief If true, the collision check will exit on first state contact. Default: true */
```

Comment:
It looks like the description needs to be updated

Reply:
Good catch, reworded to:
```C++
/** @brief The condition in which the trajectory collision check will exit. Default: FIRST */
```